src/ui/public/doc_table/doc_table.less

```diff
@@ -8,6 +8,16 @@ doc-table {
   flex: 1 1 100%;
   flex-direction: column; /* 1 */
 
+  .docTablePagerInfoMessage {
```

Comment:
Can you delete this class and add this new one instead? And can you unnest it and put it on line 3, outside of `doc-table`?

```less
.docTableBar {
  margin: 5px 5px 0;
}
```